librz/util/zip.c

```diff
@@ -24,6 +24,13 @@ static const char *gzerr(int n) {
 }
 
 RZ_API ut8 *rz_inflate(const ut8 *src, int srcLen, int *srcConsumed, int *dstLen) {
```

Comment:
Rename this as `rz_inflate_ignore_header` and add a comment in the C file saying that this is to ignore if is zlib compressed or gzipped.